civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -562,16 +562,47 @@ function civihr_employee_portal_init() {
       '/lib/sweetalert/sweetalert.min.js'
     ]);
 
+    civicrm_resources_load('org.civicrm.reqangular', ['reqangular.min.js']);
```

Comment:
I don't think this can stay like this. You are adding `reqangular.min.js` everywhere. You should add it only in the dashboard